navigation.json

```diff
@@ -120,7 +120,7 @@
           "label": "components.navigation.getInvolved.links.contribute"
         },
         "codeOfConduct": {
-          "link": "https://github.com/nodejs/admin/blob/HEAD/CODE_OF_CONDUCT.md",
+          "link": "https://github.com/nodejs/admin/blob/main/CODE_OF_CONDUCT.md",
```

Comment:
I've seen others say HEAD is more durable to default branch name changes 🤷

Reply:
Yeah, that does make sense, though I think the chances of the default branch being changed again are rather slim, and I think taking the user to the default branch rather than the head commit is a slightly better experience?

Reply:
i agree - but know others feel differently. maybe we let this PR sit for more feedback